package.json

```diff
@@ -15,10 +15,15 @@
     "sanctuary-def": "0.6.x"
   },
   "devDependencies": {
+    "browserify": "13.1.x",
     "doctest": "0.10.x",
     "eslint": "2.9.x",
     "istanbul": "0.4.x",
     "jsverify": "0.7.x",
+    "karma": "1.3.x",
+    "karma-browserify": "5.1.x",
+    "karma-mocha": "1.3.x",
+    "karma-sauce-launcher": "1.1.x",
```

Comment:
I believe the reason why that wouldn't is how karma handles plugin loading. Since we don't specify any plugins to load, it requires any compatible plugin avaiable. One way we could rectify this is doing something like which i think will work:

```
plugins: [
  require('karma-browserify'),
  require('karma-mocha'),
  require('karma-sauce-launcher'),
]
```